src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -103,6 +103,19 @@ internal static OtlpLogs.LogRecord ToOtlpLog(this LogRecord logRecord)
                 otlpLogRecord.Flags = (uint)logRecord.TraceFlags;
             }
 
+            if (logRecord.StateValues != null)
+            {
+                foreach (var stateValue in logRecord.StateValues)
+                {
+                    var attributeKey = stateValue.Key;
+                    var attributeValue = stateValue.Value?.ToString(); // TODO: Is ToString() correct here?
+
+                    // TODO: Needs validation of contract allowed into StateValues versus contract permitted for Attributes
```

Comment:
StateValues will take in just about anything. Some special cases to be known:
- .LogInformation("Price of {name} is {name}", "tomatoes", 2.99)
  - StateValues: {name: "tomatoes", name: 2.99}
  - OpenTel spec specifies that `map` types must not contain duplicates.
- .Log(LogLevel.Critical, default, ConsoleColor.Black, default, default)
  - StateValues: { "": "Black" }
  - This case seems to be allowed by the spec, but good to keep in mind as the special handling done currently by the library

I'm not sure if any special handling for these belongs here or further down on Attributes, further up (on the LogRecord which does the parsing) or even out on the exporter which exports the data. Barring direction to handle cases here, I will just transfer the data-in -> data-out.

Reply:
The same comment covers `What happens in Add() on duplicates`, since the code permits the Add and just passes the values through to the eventual exporter.

Reply:
> OpenTel spec specifies that map types must not contain duplicates.

Yes this is something we should address, but I agree it probably doesn't belong in the exporter.